lib/flutter_redux.dart

```diff
@@ -41,10 +41,13 @@ class StoreProvider<S> extends InheritedWidget {
   ///   }
   /// }
   /// ```
-  static Store<S> of<S>(BuildContext context) {
+  static Store<S> of<S>(BuildContext context, {bool listen = true}) {
```

Comment:
Woohoo! Tests look great, but I forgot about one important thing: Documentation!

Could you please add a sentence or two on the `listen` property and why it should be used? Maybe even add a short example of using it in `initState`?

Reply:
I add some sentence and example, what do you think?